pages/search.html

```diff
@@ -22,8 +22,8 @@
 				{% unless post.excluded_in_search %}
 					{% if added %},{% endif %}
 					{% assign added = false %}
-					"{{ post.url | slugify }}": {
-						"id": "{{ post.url | slugify }}",
+					"{{ post.url }}": {
```

Comment:
I think the slugify is a better (less error prone) identifier  - I don't remember the exact error but I have had issues with using a url as an index before. To address the warning message, what about just doing a check if the url is defined and only using slugify then (and do something like root if it's not).

Reply:
> just doing a check if the url is defined and only using slugify then (and do something like root if it's not).

Done 👍